src/App.vue

```diff
@@ -419,7 +412,7 @@ export default {
         AuthServices.fetchEntityInfo(this.getAuthApiUrl, this.businessId),
         this.loadBusinessInfo(),
         LegalServices.fetchTasks(this.businessId),
-        LegalServices.fetchFilings(this.businessId || this.tempRegNumber),
+        this.loadFilings(),
```

Comment:
Would it be possible to call `this.loadFilings()` from the FilingHistoryList instead of here in App.vue?  It looks like FilingHistoryList is the only component that uses the data.

Reply:
Yes but then we lose the asynchronicity of fetching all data at the same time (ie, Promise.all()).

It's true that the FHL is the only one to use the list items, but the Dashboard displays the count.

I'd rather not refactor App.vue in this ticket. Are you OK with that?

Reply:
Sure, I'll create a refactoring ticket to move call `loadFilings()` from Dashboard.vue.  As you point out, the dashboard needs the count.